testing/freerdp2/APKBUILD

```diff
@@ -0,0 +1,110 @@
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+pkgname=freerdp2
+_pkgname=FreeRDP
+pkgver=2.0.0_rc0
+_pkgver=${pkgver/_rc/-rc}
+pkgrel=0
+pkgdesc="A Remote Desktop Protocol Implementation"
+url="http://www.freerdp.com/"
+arch="all"
+options="!check"
+license="ASL-2.0"
+depends=""
+makedepends="cmake linux-headers alsa-lib-dev cups-dev eudev-dev gsm-dev
```

Comment:
as described in libfreerdp/freerdp2.pc, the only required 3rd-party library is libssl and it's included automatically to freerdp2-dev depends.

> depend = freerdp2-libs-client=2.0.0_rc0-r0
> depend = freerdp2-libs-server=2.0.0_rc0-r0
> depend = freerdp2-libs-shadow=2.0.0_rc0-r0
> depend = freerdp2-libs-winpr=2.0.0_rc0-r0
> depend = freerdp2-libs=2.0.0_rc0-r0
> depend = pc:libssl
> depend = pkgconfig

So I see no sense to add all those -dev stuff to depends_dev.